todo_app/test_viewmodel.py

```diff
@@ -0,0 +1,41 @@
+from todo_app.templates.view_model import ViewModel
+from todo_app.data.item import Item
+
+def test_view_can_ret_todo_items():
+    items= [Item('1234', 'start', 'todo')]
+    item_view_model = ViewModel(items)
+
+    todo_items = item_view_model.todo_items
+
+    assert len (todo_items) == 1
```

Comment:
It would be nice to test this with more than one item too, so we're checking it's actually filtering, at the moment replacing all your view model properties with `return self._items` would still pass the tests!